src/imars3d/backend/diagnostics/rotation.py

```diff
@@ -25,40 +24,74 @@ def find_rotation_center(
     angles: array of angles in degrees or radians, which must match the order of arrays
     in_degrees: whether angles are in degrees or radians, default is True (degrees)
     atol_deg: tolerance for the search of 180 deg paris, default is 0.1 degrees
-    ncore: number of cores to use for parallel median filtering, default is -1, which means using all available cores.
+    max_workers: number of cores to use for parallel median filtering, default is 0, which means using all available cores.
```

Comment:
can you update class docstring to match the numpy style.  For example, the arrays entry should be:

```
arrays: np.ndarray
    3D array of images, the first dimension is the rotation angle omega
```

This is because `sphinx` cannot parse the class signature to figure out the data type, so we need to provide them explicitly here.

You can see that the parameterizedFunction's docstring looks different from the one auto parsed from standard function here (https://imars3d--190.org.readthedocs.build/en/190/api/imars3d.backend.diagnostics.html#module-imars3d.backend.diagnostics.rotation).